### DISCUSSION TITLE
Request Repos for Indexing here

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/feedback#2] [Q&A] [answered]
We would love to index everything ever created, but need that would require us to watch everything you do on GitHub. You don't want that, so we would love to know:

What repositories are not available on OpenSauced? We will do our best to get them queued and ready to go.

**Comment by warrenbuckley** [comment]
@bdougie any chance you can index umbraco/umbraco-cms please

**Reply by bdougie** [reply]
Yes. Added to the queue, it will take a few hours to start showing.

**Reply by warrenbuckley** [reply]
Hiya Brian @bdougie 
It still does not show up and takes me back to this discussion page to ask for it to be added.

Do you mind checking please after the holidays.

Thanks,
Warren 😄

**Reply by bdougie** [reply]
It looks like the form is case-sensitive (not intentional). It works with `umbraco/Umbraco-CMS`.

Opened an issue to fix this. Thanks for your patience.

https://github.com/open-sauced/insights/issues/734

**Reply by brandonroberts** [reply]
Just an update here that the form is no longer case-sensitive for adding repos

**Comment by codercatdev** [comment]
@bdougie  Can we index all of `codingcatdev` repos?

**Reply by bdougie** [reply]
All repos in the org are being indexed now.

**Comment by johnstonmatt** [comment]
Hi @bdougie could you please index [polyseam/cndi](https://github.com/polyseam/cndi)? :)

**Reply by bdougie** [reply]
Hey @johnstonmatt, we are now indexing the polyseam org. Sorry for the delay. Missed this coming out of the holidays.

**Comment by fmerian** [comment]
@bdougie Hey Brian, any chance you could index [liveblocks/liveblocks](https://github.com/liveblocks/liveblocks) please? Thanks!

**Reply by bdougie** [reply]
You should see [liveblocks](https://insights.opensauced.pizza/javascript/dashboard/filter/liveblocks/liveblocks) data now.

**Comment by c0der4t** [comment]
@bdougie can you index these repos perhaps?

[ekronds/EDS_Retail](https://github.com/ekronds/EDS_Retail)

and

[c0der4t/S3-SimpleBackup](https://github.com/c0der4t/S3-SimpleBackup)

**Comment by samuelstroschein** [comment]
I request [inlang/inlang](https://github.com/inlang/inlang) :)

**Reply by bdougie** [reply]
We are moving infrastructure at the moment and testing it in staging now. This light kit happen till next week.

**Comment by nickytonline** [comment]
Can these repos be added?

- netlify/remix-compute
- netlify/remix-edge-template

**Reply by bdougie** [reply]
We won't be able to get to this till next week.

https://github.com/open-sauced/feedback/discussions/2#discussioncomment-4939066

**Comment by AyushAgnihotri2025** [comment]
Hi @bdougie could you please index [AyushAgnihotri2025/Calculator](https://github.com/AyushAgnihotri2025/Calculator)? :)

**Comment by sumitsaurabh927** [comment]
Hey, can you please add [Novu](https://github.com/novuhq/novu) to the indexing queue. It says "__The repository is not currently being indexed by OpenSauced.__" and I'd love for it to get indexed!

**Reply by bdougie** [reply]
Any repos added to an insight page will be indexed in a few hours. We have not set up notifications on this as of yet but will add it to the release notes when we do.

**Comment by bdougie** [comment, marked as answer]
All repos are now instantly added when submitted on the insight page. Depending on the size the data should start populating within a few seconds as long as they are open sourced.
